packages/lib/__tests__/formInteraction.tsx

```diff
@@ -34,6 +34,36 @@ describe('Form interaction', async () => {
     expect(fieldDriver.get.value()).toBe('batman is cool');
   });
 
+  it('should reset to values in specified in reset() arguments', async () => {
```

Comment:
Suggest:

Should reset to specific values, if they are passed as “reset” argument, like “reset({newKey: ‘newValue’})”.

Reply:
```suggestion
  it('Should reset to specific values, if they are passed as “reset” argument, like "reset({newKey: ‘newValue’})"', async () => {
```